eval/eval.py

```diff
@@ -824,18 +824,21 @@
     "Z003": ["upper4lower-common"],
     "Z004": ["upper4lower-common"],
     "Z005": ["upper4lower-common"],
+    "Z005/w": ["upper4lower-common"],
     "Z006": ["lower4upper-acro"],
-    # "E001" : ["XXX"],
-    # "E002" : ["XXX"],
-    # "E003" : ["XXX"],
+    "E001" : ["XXX"],
```

Comment:
What does the XXX stand for? Could it be made more descriptive, such as "[No corresponding IEC category]" or something like that?

Reply:
'XXX' was just a placeholder, changed to your suggestion.